watchdog.js

```diff
@@ -59,7 +60,7 @@ function executeScript () {
     return
   }
   console.log('in execute')
-  child = cp.fork(SCRIPT_PATH, [platform])
+  child = cp.fork(SCRIPT_PATH, [platform, model])
```

Comment:
Have to be careful here because old systems won't have this update until after the update. So the first update package will not be supplied the ``model`` param.

Reply:
right, model is being defaulted to gaia since every upboard+sintra will have this set